A.D./src/test/java/com/adoptdifferently/model/UserTest.java

```diff
@@ -1,5 +1,85 @@
 package com.adoptdifferently.model;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.adoptdifferently.model.User;
+import com.adoptdifferently.model.CatListing;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@SuppressWarnings("unused")
 public class UserTest {
+    private User user;
+
+    @BeforeEach
+    public void setUp() {
+        user = new User("John", "john@example.com", "123456789", "password", "USA", "New York", "10001", 1);
```

Comment:
You seem to repeat this code quite a bit in the test. Because you preformed setup before just call user it will make your code more DRY. And make it more readable.